### DISCUSSION TITLE
How to cite floris?

### DISCUSSION THREAD

**Discussion by ghost** [NatLabRockies/floris#1046] [General]
Hey,

I might have missed it somewhere in the docs, but does there exist a standard 'how to cite' bibtex snippet for Floris?

If not, is there a preferred 'main' article to refer to? 

Best,
Marcus

**Comment by paulf81** [comment]
Thank you for asking this question.  I believe we used to have a 'how to cite' section in the documentation with a preferred bibtext snippet, but it seems to have slipped out.  I will open a new issue to decide on the current preferred citation and re-include in the README and/or docs

**Comment by misi9170** [comment]
Linking the open issue here: #1047 

In the meantime, you could use something like the following:

NREL, "FLORIS. Version 4.2.1," _GitHub repository_, 2024. [Online]. Available: https://github.com/NREL/floris

or, as a bibtex entry, something like
```
@Article{FLORIS_v421,
author = {NREL},
title = {{FLORIS}. Version 4.2.1},
year = {2024},
publisher = {GitHub},
journal = {GitHub repository},
url = {https://github.com/NREL/floris}
}
```

**Comment by ghost** [comment]
Hey, thanks for the replies! 

I could see that people were citing different papers when using floris, so I definedly think having a "unified" way of doing it makes sense. 

I will just use the Bibtex snippet you provided; that will do perfectly fine for my case!
